packages/components/src/popover/index.tsx

```diff
@@ -223,6 +228,30 @@ const UnforwardedPopover = (
 		shouldShift = __unstableShift;
 	}
 
+	if ( anchorRef !== undefined ) {
+		deprecated( '`anchorRef` prop in Popover component', {
```

Comment:
I believe this is the standard notation we should be using:

```suggestion
		deprecated( '`anchorRef` prop in wp.components.Popover', {
```

Reply:
Good point. I've updated all deprecation warning in the Popover component to follow the suggested notation

https://github.com/WordPress/gutenberg/pull/43691/commits/dede8d981a31b74421ffa8fac92d78be87130adb